default/decorators.py

```diff
@@ -41,3 +41,42 @@ def wrap(request, *args, **kwargs):
         return function(request, *args, **kwargs)
 
     return wrap
+
+
+def split_size(function):
```

Comment:
Can we change the name of this method to `validate_split_size`?

Reply:
Done.